### ISSUE TITLE
Results reproduce - PointMass2D Dense - Extra Obstacles

### ISSUE THREAD

**Issue by changchencc** [joaoamcarvalho/mpd-public#2]
Hi @jacarvalho, thanks for releasing the codes!

I ran the inference.py with the pre-trained model on the PointMass2D Dense task (EnvDense2D-RobotPointMass). I can only get a success rate of around 71%.  You mentioned that two hyperparams, 'weight_grad_cost_collision' and 'weight_grad_cost_smoothness', might need to be changed. Could you be more specific on that and share us the range to search these two values? 
And, other than that, are there other things I need to take care of to reproduce the results?

Thank you!

**Comment by joaoamcarvalho** [comment]
Hey, 
The paper results are hard to reproduce with the current code version, as we introduced a couple of changes since the IROS submission (e.g., the training data is slightly different), and the success rate was computed by averaging over 100 seeds. At the current version, I'd take the code more as a starting point.
We are working on a better version of the code (just hang on for a bit) and the hyperparameter search, but unfortunately, I did not have too much time to work on this.
Nevertheless, the parameter ranges we are testing are roughly 
weight_grad_cost_collision: [1e-2, 1e-1]
weight_grad_cost_smoothness: [1e-8, 1e-6]
